ChooseArea.svelte

```diff
@@ -89,7 +89,11 @@
   loadAuthorities();
 
   function start() {
-    if (inputList.value) {
+    let auth_names = Array.from(dataList.childNodes.values()).map(function (x) {
```

Comment:
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/Arrow_functions may be more readable here: `(x) => x.value`

Reply:
Resolved in 06ff173